alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -324,8 +324,12 @@ protected function getLivePackagerUrl($serverNode, $streamFormat = null)
 		
 		$livePackagerUrl = "$livePackagerUrl/p/$partnerID/e/$entryId/sd/$segmentDuration/";
 		$secureToken = $this->generateLiveSecuredPackagerToken($livePackagerUrl);
-		$livePackagerUrl .= "t/$secureToken/"; 
-		
+		$livePackagerUrl .= "t/$secureToken/";
+
+		$entry = $this->getDynamicAttributes()->getEntry();
+		if ($entry->getExplicitLive() &&  (!$entry->canViewExplicitLive()))
```

Comment:
@kalturaguy will this require changes on the packager configuration file as well?

As to passing "type/user" it would be better to always pass type if explicitLive is turned on rather than passing it only if its type user. So if user pass user and if admin pass admin.

Aslo maske "user" & "admin" const's.

Reply:
@yossipapi  yes, we are modifying the packager config too...

Reply:
I looked at the liveController code, since we only set a clipTo value if explicit live is enabled then even if we pass /type/user when it is off it won't have any effect.
I will make the requested change.